packages/salesforcedx-apex-replay-debugger/src/messages/i18n.ts

```diff
@@ -32,5 +32,7 @@ export const messages = {
   fetching_heap_dump:
     'Retrieving heap dump from server. Heap dump information=%s',
   malformed_log_line:
-    "Encountered a malformed HEAP_DUMP log line, skipping. Log line number=%d, log line='%s'."
+    "Encountered a malformed HEAP_DUMP log line, skipping. Log line number=%d, log line='%s'.",
+  reconcile_heapdump_error:
+    'Problems were encountered while using heapdump: %s.'
```

Comment:
A catch-all message in case something goes wrong with parsing the heapdump. The user should probably remove the ApexExecutionOverlayResult object from their org.

Reply:
This is very minor but we should probably use "heap dump", two words, to be consistent with the rest of the messages.

Reply:
Thanks for suggesting what the user should do next, @DatGuyJonathan . Would there be more than one ApexExecutionOverlayResult in the org at a time? If so, how would the user know which one to delete? 

Yes, "heap dump" should be two words.

Reply:
@ruthemmanuelle , they can have more than one ApexExecutionOverlayResult in the org, but the error will only occur for one ApexExecutionOverlayResult. We have the ID of the ApexExecutionOverlayResult object so we can possibly suggest that for their query.

Reply:
Ok, cool. How about something like this? (I assumed the %s in your original message was a description of the problems, but if not then we can eliminate it from the first sentence of my suggested message.)

'Problems were encountered while using a heap dump: %s. Use Tooling API to delete the ApexExecutionOverlayResult record with the ID %s: From a terminal, run "sfdx force:data:record:delete -t -s ApexExecutionOverlayResult -i %s".'